src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './components/TodoInfo';
+import { User } from './components/UserInfo';
+import users from './api/users';
+import todos from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const usedTodos: Todo[] = todos.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+function getMaxTodoId(tasks: Todo[]) {
+  const todoIds = [];
+
+  for (let i = 0; i < tasks.length; i += 1) {
+    todoIds.push(tasks[i].id);
+  }
+
+  return todoIds.sort((task1, task2) => task2 - task1)[0];
```

Comment:
let's name your variables differently than `task1`, `task2` to make your code more clear and understandable, go for `firstTask` and `secondTask`